src/components/navigation-bar.component.tsx

```diff
@@ -17,25 +25,88 @@ const useStyles = makeStyles(theme => ({
   title: {
     flexGrow: 1,
   },
+  list: {
+    width: 250,
+  },
+  formControl: {
+    margin: theme.spacing(3),
+  },
 }))
 
 export function NavigationBarComponent() {
   const { t } = useTranslation()
   const classes = useStyles()
+  const [state, setState] = React.useState({
+    top: false,
+    left: false,
+    bottom: false,
+    right: false,
+    langauge: '',
+  })
```

Comment:
Please split `language` in a different state variable.  
Also there is no need to keep state for all 4 sides drawer, because we use only one side.  
We need just one boolean variable.